packages/node_modules/@webex/internal-plugin-conversation/test/unit/spec/decrypt-transforms.js

```diff
@@ -65,6 +65,138 @@ describe('plugin-conversation', () => {
       });
     });
 
+    describe('reactions', () => {
+      it('calls the correct method if a reaction2Summary object is passed in', () => {
+        const transform = transforms.find((t) => t.name === 'decryptObject');
+        const transformStub = sinon.stub();
+
+        const ctx = {
+          transform: transformStub
+        };
+        const key = null;
+        const activity = {
+          objectType: 'reaction2Summary'
+        };
+
+        transform.fn(ctx, key, activity);
+
+        assert.equal(transformStub.lastCall.args[0], 'decryptReaction2summary');
+      });
+
+      it('decrypts the activity object and its children and its reactions', () => {
+        const transform = transforms.find((t) => t.name === 'decryptActivity');
+        const transformStub = sinon.spy();
+
+        const ctx = {
+          transform: transformStub
+        };
+        const key = null;
+        const activity = {
+          objectType: 'activity',
+          verb: 'post',
+          encryptionKeyUrl: 'keyUrl1',
+          object: {
+            objectType: 'comment'
+          },
+          children: [
+            {
+              verb: 'add',
+              objectType: 'activity',
+              encryptionKeyUrl: 'keyUrl1',
+              object: {
+                objectType: 'reaction2Summary',
+                reactions: [
+                  {displayName: 'reaction1'},
+                  {displayName: 'reaction2'}
+                ]
+              }
+            },
+            {
+              verb: 'add',
+              objectType: 'activity',
+              encryptionKeyUrl: 'keyUrl1',
+              object: {
+                objectType: 'reaction2Summary',
+                reactions: [
+                  {displayName: 'reaction1'},
+                  {displayName: 'reaction2'}
+                ]
+              }
+            }
+
+          ]
+
+        };
+
+        return transform.fn(ctx, key, activity)
```

Comment:
i couldn't figure out a way to get spy/assert on the recursive behavior here. any advice is welcome!